wao/error_watcher.py

```diff
@@ -41,14 +43,13 @@ def smooth_romeo_restart(error_line):
     is_romeo_alive = romeo is not None
     error_line = "[REPORT TO TRELLO]" + error_line
     error_line += (" [SENDING SS]" if is_romeo_alive else " [POOL EMPTY. NO ROMEO FOUND]")
-    post_request(error_line) #todo remove
 
     if is_romeo_alive:
         romeo.perform_sell_signal(RomeoExitPriceType.SS)
-        # romeo.send_error_report(error_line) #todo
-    # else:
-    #     post_request(error_line) #todo uncomment
-    #     send_to_trello(error_line) #todo title-error_line, description-error_line
+        romeo.send_error_report(error_line)
+    else:
+        post_request(error_line)
```

Comment:
can you use notifier.post_request(error_line, is_from_error_report=True) 

use same notifier object as line 93

Reply:
done

Reply:
done.